src/shared/views/visit/VisitRoute.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import { Visit } from './index'
+
+const VisitRoute = () => {
+  return (
+    <div>
+      <Visit />
```

Comment:
I would think that `<Visit>` should just be `VisitRoute`. Not sure why they would be seperate?

also FYI if you just have on component you don't need to wrap it in a `div`